src/components/BrowseSong/BrowseSong.jsx

```diff
@@ -0,0 +1,52 @@
+import { Col, Container, Row } from "reactstrap";
+import SongCard from "../Cards/SongCard";
+import { data } from "../../assets/json/data";
```

Comment:
use json-server to host mock api instead of importing the data directly from .json file

Reply:
Will provide data